File: beginner/c/2540.c
#include <stdio.h>

int main() {
    int N, voto;
    
    while (scanf("%d", &N) == 1) {
        int votos_favoraveis = 0;
        
        for (int i = 0; i < N; i++) {
            scanf("%d", &voto);
            votos_favoraveis += voto;
        }
        
        if (votos_favoraveis * 3 >= 2 * N)
            printf("impeachment\n");
        else
            printf("acusacao arquivada\n");
    }

    return 0;
}

File: beginner/java/2540.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        int N, voto;

        N = scanner.nextInt();
    
        while (scanner.hasNext()) {
            int votos_favoraveis = 0;
            
            for (int i = 0; i < N; i++) {
                voto = scanner.nextInt();
                votos_favoraveis += voto;
            }
            
            if (votos_favoraveis * 3 >= 2 * N)
                System.out.println("impeachment");
            else
                System.out.println("acusacao arquivada");
        }
    }
}
